compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TileAttention.cpp

```diff
@@ -415,7 +416,8 @@ void convertToOnlineAttention(IREE::LinalgExt::AttentionOp attnOp,
   OnlineAttentionOp onlineAttn = rewriter.create<OnlineAttentionOp>(
       loc, TypeRange{accFill.getType(), maxFill.getType(), sumFill.getType()},
       attnOp.getQuery(), attnOp.getKey(), attnOp.getValue(), attnOp.getScale(),
-      accFill, maxFill, sumFill, rewriter.getAffineMapArrayAttr(indexingMaps));
+      attnOp.getMask() ? attnOp.getMask() : Value(), accFill, maxFill, sumFill,
```

Comment:
Can you hoist this ternary statement out?

Reply:
Alrighty